ndn-svs/svspubsub.hpp

```diff
@@ -56,16 +56,37 @@ class SVSPubSub : noncopyable
   virtual
   ~SVSPubSub() = default;
 
-  /** Subscription Data type */
-  struct SubscriptionPacket
+  struct SubscriptionData
   {
-    const Data& data;
+    /** @brief The name of the received publication */
+    const Name& name;
+
+    /** @brief Raw buffer of the received data */
+    const uint8_t* data;
+
+    /** @brief Length of the received data */
+    const size_t length;
+
+    /** @brief Block of received data packet
```

Comment:
based on what I'm seeing this is actually just the *content* block, aka the payload, not the whole data packet.

Reply:
This was somewhat sloppy. Changed everything to exclusively use span now.